underscore.js

```diff
@@ -1100,6 +1100,9 @@
 
    // Return a copy of the object without the blacklisted properties.
   _.omit = restArgs(function(obj, keys) {
+    if(obj != null && Object.getOwnPropertySymbols(obj) != null){
```

Comment:
Regardless, FF and Chrome both added support only about a year ago, which is typically less than our support window. I think it's less of an issue in evergreen browsers (Edge is evergreen, right?) but probably something we should figure out in another thread.

Regardless, @edwardkerry maybe add a check that `Object.ownPropertySymbols` exists before using it?